src/librustc_error_codes/error_codes.rs

```diff
@@ -628,4 +628,5 @@ E0766: include_str!("./error_codes/E0766.md"),
     E0755, // `#[ffi_pure]` is only allowed on foreign functions
     E0756, // `#[ffi_const]` is only allowed on foreign functions
     E0757, // `#[ffi_const]` functions cannot be `#[ffi_pure]`
+    E0768, // `Foo { a, b }` pattern  instead of correct `Foo(a, b)`.
```

Comment:
This is strange, I recall us being at `E0768` already... Well, if there are no conflicts. :)